packages/overlay/src/telemetry/utils/traces.ts

```diff
@@ -1,4 +1,4 @@
-import { log } from "../../../lib/logger";
+import { log } from "~/lib/logger";
```

Comment:
Pretty sure we should look into switching to a proper logging library or just use Sentry and Spotlight itself to debug spotlight.